synanno/templates/opendata.html

```diff
@@ -40,306 +40,204 @@ <h2 class="mb-3">Welcome to SynAnno Annotation Tool</h2>
-        >Materialization (url/name of the table containing the ID association
-        and bounding box information)</label
-      >
-      <p>
-        <input
-          class="form-control"
-          type="text"
-          id="materialization_url"
-          name="materialization_url"
-          placeholder="Materialization - synapses"
-          value="h01/synapse-export_000000000000.csv"
-        />
-      </p>
-      <label for="secrets_file" class="form-label"
-        >Secrets (File containing the bucket secrets, defaults to looking in
-        ~/.cloudvolume/secrets)</label
-      >
-      <p>
-        <input
-          class="form-control"
-          type="file"
-          id="secrets_file"
-          name="secrets_file"
-        />
-      </p>
-      <label class="form-label">SynAnno JSON file</label>
-      <p>
-        <input
-          class="form-control {{mode}}"
-          type="file"
-          id="formFile"
-          name="file_json"
-        />
-      </p>
-
-      <label class="form-check-label">
-        <input
-          type="radio"
-          class="form-check-input"
-          id="view_centric_button"
-          name="view_style"
-          value="view"
-          {%
-          if
-          (view_style=="view")
-          %}
-          checked
-          {%
-          endif
-          %}
-        />
-        View Centric
-      </label>
-      <label class="form-check-label">
-        <input
-          type="radio"
-          class="form-check-input"
-          id="neuron_centric_button"
-          name="view_style"
-          value="neuron"
-          {%
-          if
-          view_style=="neuron"
-          %}
-          checked
-          {%
-          endif
-          %}
-        />
-        Neuron Centric
-      </label>
-      <label class="form-label">
-        <select id="coordinates" name="coordinates">
-          <option value="xyz" selected>XYZ</option>
-          <option value="xzy">XZY</option>
-          <option value="zyx">ZYX</option>
-          <option value="zxy">ZXY</option>
-          <option value="yxz">YXZ</option>
-          <option value="yzx">YZX</option>
-        </select>
-        Coordinate Order
-      </label>
-
-      <div class="mb-1"></div>
-
-      <div
-        id="neuron-form"
-        {%
-        if
-        view_style=="view"
-        %}
-        class="hidden-form"
-        {%
-        endif
-        %}
-      >
-        <label for="pre-post-syn" class="form-label"
-          >Pre- and postsynaptic IDs</label
-        >
-        <div class="number-container" id="pre-post-syn">
-          <input
-            class="number-dims"
-            type="number"
-            id="preid"
-            name="preid"
-            placeholder="Presynaptic ID"
-          />
-          <input
-            class="number-dims"
-            type="number"
-            id="postid"
-            name="postid"
-            placeholder="Postsynaptic ID"
-          />
+    <div class="accordion">
```

Comment:
I like the idea of using Bootstrap's accordion class. However, I tried fitting every thing onto a single page such that the user does not has to scroll. How do you feel about collapsing the Secrets and Configurations tap at the start?